app/players/player.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> int:
+        pass
+
+    def player_info(self) -> str:

```

Comment:
The `player_info` method should be declared as an abstract method. Add the `@abstractmethod` decorator above this method to ensure all subclasses implement it.